scripts/blockchain

```diff
@@ -13,22 +13,35 @@ def main():
     blockchain_path = pathlib.Path(__file__).absolute().parent.parent.joinpath("blockchain")
     blockchain_node_path = blockchain_path.joinpath("node_modules")
     blockchain_json_src_path = blockchain_node_path.joinpath("singularitynet-alpha-blockchain")
-    token_json_src_path = blockchain_node_path.joinpath("singularitynet-token-contracts", "SingularityNetToken.json")
+    token_json_src_dir = blockchain_node_path.joinpath("singularitynet-token-contracts")
```

Comment:
both `blockchain_json_*` and `token_json_*` are directories now, so even more reason to keep their names consistent